libpod/container_internal.go

```diff
@@ -830,6 +830,12 @@ func (c *Container) init(ctx context.Context) error {
 	if err := c.save(); err != nil {
 		return err
 	}
+	if c.config.HealthCheckConfig != nil {
+		if err := c.CreateTimer(); err != nil {
```

Comment:
CreateTimer shouldn't be public - I don't see anyone outside of libpod ever wanting to do this themselves